plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/internal/core/OpenShiftCoreActivator.java

```diff
@@ -43,12 +48,39 @@ public static BundleContext getBundleContext() {
 	    return myContext;
 	}
 
+	@Override
     public void start(BundleContext context) throws Exception {
         super.start(context);
         myContext = context;
+        String[] connections = OpenShiftPreferences.getInstance().getConnections();
```

Comment:
i would definitely move the code that's responsible to add serialized connections to the registry to its own method.